src/main/kotlin/it/pagopa/wallet/controllers/WalletController.kt

```diff
@@ -106,6 +106,28 @@ class WalletController(
         return walletService.findWalletByUserId(xUserId).map { ResponseEntity.ok(it) }
     }
 
+    @SuppressWarnings("kotlin:S6508")
```

Comment:
why this suppression? add reason as comment, see an example [here](https://pagopa.atlassian.net/wiki/spaces/I/pages/615449318/Developer+Style+Guide#Disattivare-gli-avvisi-di-Sonarcloud)

Reply:
Fixed with 4808ab08b68cf1410db0876cb38c53a7e3941b69